### ISSUE TITLE
Enable romajization of kana

### ISSUE THREAD

**Issue by GoogleCodeExporter** [melink14/rikaikun#71]
```
New (minor) feature:
Please make an optional mode where hiragana/katakana particles is written as 
romaji as well, would be very useful in order to learn these.
Maybe too hard for kanji, but at least do it for the others.

Thanks!
```

Original issue reported on code.google.com by `jonatan....@gmail.com` on 24 Jan 2012 at 11:28

**Comment by melink14** [comment]
I think this is reasonable though I'm sure there will be disagreement into what romanization style to use.

**Comment by aensaro** [comment]
Please add this, at least Hepburn as it's the most used method.

**Comment by melink14** [comment]
https://wanakana.com/

**Comment by tora-pan** [comment]
@melink14 Do we still want to do this? I'm all for working on the feature but if I may share my two cents as well...

Romaji is great for the first week or two of Japanese learning. Most of the resources on the web with a targeted audience of "Japanese Learner" almost assume that they know ひらがな and カタカナ. 

When you constantly rely on using romaji, it really holds back your Japanese IMO.

When reading things on Japanese websites, watching things on TV, walking around the streets of downtown 渋谷, you aren't going to see *romaji* anywhere.

**Comment by melink14** [comment]
I don't think this is worth doing unless you personally were interested in it. Certainly, I don't think it's very high priority.

Either way it would definitely be a setting that was default set to 'off' (I really should do a post update popup with new changes so people can find stuff; and also post via rikaikun's twitter)

That said, while I agree that romaji is not that useful for learners it could be helpful for people who aren't trying to learn but want to pronounce Japanese words.

Given we can use a library for conversion adding at the end like you showed seems fine to me.
